### PR TITLE
reg table button update

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#9108

*Description of changes:*
- see ticket for details

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/tests/unit/test-data/mock-registration-responses.ts [hunk 1]
```diff
@@ -45,6 +45,21 @@ export const mockedRegistration2Child: RegistrationSummaryIF = {
   securedParties: 'Bank of Montreal'
 }
 
+export const mockedRegistration3: RegistrationSummaryIF = {
+  baseRegistrationNumber: '003423B',
+  clientReferenceId: '',
+  createDateTime: '2019-08-20T17:21:17+00:00',
+  expireDays: '0',
```
Comment (jordiwes) [inline ppr-ui/tests/unit/test-data/mock-registration-responses.ts]: by the time I merge my PR, this will be a number :)

**Review by jordiwes — APPROVED** [review]
Looks good, just the one comment

File: ppr-ui/tests/unit/test-data/mock-registration-responses.ts (hunk 1 above)
Comment (kialj876) [reply ppr-ui/tests/unit/test-data/mock-registration-responses.ts]: haha okay I will update

Comment (kialj876) [reply ppr-ui/tests/unit/test-data/mock-registration-responses.ts]: updated

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**